### PR TITLE
Allow Symfony 4.0

### PR DESCRIPTION
**Pull request by nicolas-grekas**

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
Can anyone merge this now please? Blocking SF4 CI also.

**Comment by alcaeus** [comment]
@nicolas-grekas can you look at the failing tests? Happy to merge once it's stable.

File: Tests/DependencyInjection/AbstractDoctrineCacheExtensionTest.php [hunk 1]
```diff
@@ -219,7 +219,7 @@ public function testAliasesCache()
             $this->assertTrue($container->hasDefinition($key));
 
             foreach ($aliases as $alias) {
-                $this->assertEquals(strtolower($key), (string) $container->getAlias($alias));
+                $this->assertEquals(strtolower($key), strtolower($container->getAlias($alias)));
```
Comment (nicolas-grekas) [inline Tests/DependencyInjection/AbstractDoctrineCacheExtensionTest.php]: ids are case sensitive in 3.3, and case insensitive (ie lowercased) before.

**Comment by nicolas-grekas** [comment]
test are green, ready to merge :)

**Comment by nicolas-grekas** [comment]
ping @alcaeus

**Comment by alcaeus** [comment]
Sorry for the delay - gotta have lunch breaks 🍔

**Pull request merged**